<launch>
	<arg name="veh" doc="Name of vehicle" />

	<remap from="vehicle_avoidance_control_node_test/car_cmd_out" to="vehicle_avoidance_control_node/car_cmd_in" />
	<remap from="vehicle_avoidance_control_node_test/vehicle_detected" to="vehicle_avoidance_control_node/vehicle_detected" />
	<remap from="vehicle_avoidance_control_node_test/vehicle_pose" to="vehicle_avoidance_control_node/vehicle_pose" />
	<remap from="vehicle_avoidance_control_node_test/car_cmd_in" to="vehicle_avoidance_control_node/car_vel" />

	<include file="$(find vehicle_detection)/launch/vehicle_avoidance_control_node.launch">
		<arg name="veh" value="$(arg veh)"/>
		<arg name="local" value="true" />
	</include>

	<group ns="$(arg veh)">
        <node name="vehicle_avoidance_control_node_test" pkg="vehicle_detection" type="vehicle_avoidance_control_node_test.py" output="screen" />
	</group>
</launch>
